pgcli/packages/parseutils.py

```diff
@@ -197,7 +197,8 @@ def extract_tables(sql):
     # to have is_function=True
     identifiers = extract_table_identifiers(stream,
                                             allow_functions=not insert_stmt)
-    return tuple(identifiers)
+    # In the case 'sche.<cursor>', we get an empty TableReference; remove that
```

Comment:
Could you add a quick test for this behavior in test_parseutils to prevent a regression?